frontend/app/src/composables/filter-paginate.ts

```diff
@@ -202,7 +204,29 @@ export const usePaginationFilters = <
     {
       immediate: false,
       resetOnExecute: false,
-      delay: 0
+      delay: 0,
+      onError(e) {
+        const { notify } = useNotificationsStore();
+        const { t } = useI18n();
```

Comment:
Shouldn't we pull this on the upper composable scope? 
I am thinking if there is a chance that these two run outside of the right context and failing.

Reply:
ok, will move them up